MANIFEST.in

```diff
@@ -7,7 +7,6 @@ include README.rst
 include README
 include tox.ini
 include setup.py
-include elasticsearch/connection/esthrift/Rest-remote
```

Comment:
This file doesn't seem to exist anymore so shouldn't be included in the manifest?

Reply:
definitely not needed, nor wanted, it is an artifact from darker times.